client/src/components/Submissions/Submissions.jsx

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import { useHistory, useLocation, Link } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+import { Button } from '_common';
+import { FileInputDropZone, Section } from '_common';
+import { useSystems } from 'hooks/datafiles';
+import { useUpload } from 'hooks/datafiles/mutations';
+import DataFilesUploadModalListingTable from '../DataFiles/DataFilesModals/DataFilesUploadModalListing/DataFilesUploadModalListingTable.jsx';
+import styles from './Submissions.module.scss';
+
+const SubmissionsUpload = () => {
+  const history = useHistory();
+  const location = useLocation();
+  const reloadCallback = () => {
+    history.push(location.pathname);
+  };
+
+  const { data: allSystems } = useSystems();
+  const submissionSystem = allSystems.find((s) => {
+    return s.system && s.system.startsWith('apcd.submissions');
```

Comment:
We could also change the `scheme` of the submission system in `_PORTAL_DATA_DEPOT_LOCAL_STORAGE_SYSTEMS` to be `submissions`, and return the system with that scheme